.github/workflows/docker-release.yml

```
@@ -0,0 +1,52 @@
+name: Daily build
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every day at 1:00 AM
+  workflow_dispatch:
+jobs:
+  release_image:
+    name: Build and publish docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_REPOSITORY: mongodb/atlas
+      PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Set date
+        id: set-date
+        run: |
+          DATE=$(date +'%Y-%m-%d')
```

IMHO using timestamp would be safer unless we intentionally wanty  to override images daily?
```suggestion
          DATE=$(date +'%Y-%m-%d')
```